gnucash/report/standard-reports/transaction.scm

```diff
@@ -1867,9 +1886,15 @@ tags within description, notes or memo. ")
 
 ;; Define the report.
 (gnc:define-report
+ 'version 2
```

Comment:
Why did you change the version number ? I'm not sure what the side-effects are in general.

Reply:
personal pride :) will revert.